docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -227,87 +227,153 @@ sudo -E printenv
+elif [ "$os" == "linux" ]; then
+    if ! command -v apt > /dev/null 2>&1; then
+        echo "apt is not installed; install apt and try again"
+        exit 1
+    fi
+    apt-get update
+    apt-get install -y mailutils msmtp msmtp-mta
+    echo "tls_trust_file /etc/ssl/certs/ca-certificates.crt" >> ~/.msmtprc
+else
+    echo "Unsupported OS: $os. This script expects Darwin or Linux."
+    exit 1
 fi
 
-if [ "$os" == "linux" ]
-then
-	sudo apt-get install mailutils
-	sudo apt-get install msmtp
-	sudo apt-get install msmtp-mta
-fi
+cat <<EOF >> ~/.msmtprc
+logfile ~/.msmtp.log
+account $CS_ACCOUNT
+host $CS_HOST
+port $CS_PORT
+from $CS_EMAIL
+user $CS_USERNAME
+password $CS_PASSWORD
+account default: $CS_ACCOUNT
+EOF
+
+# Restrict permissions for the .msmtprc file to avoid security issues
+chmod 600 ~/.msmtprc
+
+# Export mail variables to the shell environment
+shell_rc=~/.$(basename $SHELL)rc
+{
+    echo "export MAIL_SERVER=$CS_HOST"
+    echo "export MAIL_PORT=$CS_PORT"
+    echo "export MAIL_USE_TLS=$CS_USE_TLS"
+    echo "export MAIL_USE_SSL=$CS_USE_SSL"
+    echo "export MAIL_USERNAME=$CS_EMAIL"
+    echo "export MAIL_PASSWORD=$CS_PASSWORD"
+} >> "$shell_rc"
+
+# Send email
+echo "From: $CS_EMAIL_FROM
+To: $CS_EMAIL_TO
+Subject: $CS_EMAIL_SUBJECT
+$CS_EMAIL_BODY" | msmtp --debug --from=$EMAIL_ -t $CS_EMAIL_TO
 
-if [ "$os" == "darwin" ]
-then
-	brew install mailutils
-	brew install msmtp
-	echo "set sendmail=/usr/local/bin/msmtp" | sudo tee -a /etc/mail.rc
-fi
+```
 
-echo "defaults
-auth on
-tls on" >> ~/.msmtprc
-
-if [ "$os" == "linux" ]
-then
-	echo "tls_trust_file /etc/ssl/certs/ca-certificates.crt" >> ~/.msmtprc
-elif [ "$os" == "darwin" ]
-then
-	{ echo -n "tls_fingerprint " &
-	  msmtp --serverinfo --tls --tls-certcheck=off --host=smtp.gmail.com --port=587 \
-	  | egrep -o "([0-9A-Za-z]{2}:){31}[0-9A-Za-z]{2}" ;
-	} >> ~/.msmtprc
-fi
+:::info Script Execution
 
-echo "logfile ~/.msmtp.log
-account gmail
-host smtp.gmail.com
-port 587
-from $EMAIL_
-user $USERNAME_
-password $PASSWORD_
-account default: gmail" >> ~/.msmtprc
-
-echo "export MAIL_SERVER = smtp.gmail.com
-export MAIL_PORT = 587
-export MAIL_USE_TLS = True
-export MAIL_USE_SSL = False
-export MAIL_USERNAME = $EMAIL_
-export MAIL_PASSWORD = $PASSWORD_" >> ~/.$(basename $SHELL)rc
+This script is written in `Bash`. When running with **Custom Script**, you need to set the `Execute With` parameter as `Bash`. For more information, please visit the [**Custom Script Input Variables**](/workflows/common-workflow-steps/custom-script#input-variables) documentation section.
 
-```
+:::
+
+:::tip Reciever Email Address
+
+If you want to send an e-mail to multiple e-mail addresses instead of a single e-mail address. In the `CS_EMAIL_TO` parameter, it will be enough to write all the addresses to send an e-mail separated by commas. For example; `CS_EMAIL_TO=example@email.com,example2@email.com`
```

Comment:
At the beginning of the section, the word was written as `email`, but here we see a different style for the same concept. `e-mail`

Both are correct. ([ref](https://eslbuzz.com/e-mail-or-email/))

On the other hand, **we should have a single way of writing in the section for the same concepts.** Choose one and stick with that.